instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/CamelEventNotifier.java

```diff
@@ -59,7 +59,7 @@ private void onExchangeSending(ExchangeSendingEvent ese) {
 
     String name =
         sd.getOperationName(ese.getExchange(), ese.getEndpoint(), CamelDirection.OUTBOUND);
-    Span span = CamelTracer.TRACER.startSpan(name, sd.getInitiatorSpanKind());
+    Span span = Span.fromContext(CamelTracer.TRACER.startSpan(name, sd.getInitiatorSpanKind()));
     sd.pre(span, ese.getExchange(), ese.getEndpoint(), CamelDirection.OUTBOUND);
     ActiveSpanManager.activate(ese.getExchange(), span, sd.getInitiatorSpanKind());
     CamelPropagationUtil.injectParent(Context.current(), ese.getExchange().getIn().getHeaders());
```

Comment:
would be good to use the context returned from `startSpan` in activate(), e.g.
```suggestion
    Context context = CamelTracer.TRACER.startSpan(name, sd.getInitiatorSpanKind());
    Span span = Span.fromContext(context);
    sd.pre(span, ese.getExchange(), ese.getEndpoint(), CamelDirection.OUTBOUND);
    ActiveSpanManager.activate(ese.getExchange(), context, sd.getInitiatorSpanKind());
    CamelPropagationUtil.injectParent(context, ese.getExchange().getIn().getHeaders());
```